playbook-website/app/views/pages/visual_guidelines_partials/_pb_doc_shadow.html.erb

```diff
@@ -0,0 +1,82 @@
+<%= pb_rails("title", props: {margin_top: "xl", text: "Shadows", tag: "h1", size: 1 }) %>
+
+<%= pb_rails("body", props: { margin_y: "md"} ) do %>
+  Shadows are a night way of showing depth. Modals, buttons, & popovers are examples of items that need to have a shadow to indicate their position relative to others. Our tokens help you expand those to the other custom elements you build using playbook.
+<% end %>
+
+<br>
+
+<%= pb_rails("title", props: { margin_top: "lg", text: "Tokens", tag: "h4", size: 4 }) %> 
+
+<%= pb_rails("body", props: { margin_bottom: "sm", margin_top: "xs" } ) do %>
+  Make your own styles using Playbook tokens to keep your site consistent.
+<% end %>
+
+    <%= pb_rails("card", props: {shadow: "deep", margin_bottom: "xl", padding: "none", padding_top: "none" }) do %>
+
+  <%= pb_rails("flex", props: {orientation: "column", align: "stretch", justify: "center", padding_x: "xl", padding_y: "md"}) do %>
+      
+            <%= pb_rails("card", props: {shadow: "deep", padding: "none"}) do %>
+              <%= pb_rails("flex", props: { wrap: true, justify: "between", align: "center" }) do %>
+                <%= pb_rails("body", props: {classname: "flex-item", margin_right: "md" }) do %>
+                  <%= pb_rails("flex", props: { wrap: true, align: "center" }) do %>
+                    <%= pb_rails("circle_icon_button", props: { variant: "link", icon: "copy", padding_right: "xs" }) %>
+                    $shadow_deep
+                    <% end %>
+                <% end %>
+             
+                <%= pb_rails("body", props: {classname: "flex-item", padding_right: "sm", margin_left: "sm" }) do %>
+                  <%= pb_rails("caption", props: { text: "value: 0 4px 10px 0 rgb(60 106 172 / 16%)", size: "xs", padding_right: "xs" }) %>
+                  <% end %>
+            <% end %>
+            <% end %>
+            
+            <br>
+            
```

Comment:
Take a look at these files:
https://github.com/powerhome/playbook/blob/master/playbook-website/app/views/pages/visual_guidelines_partials/_border_radius.html.erb
https://github.com/powerhome/playbook/blob/master/playbook-website/app/views/pages/visual_guidelines_partials/_max_width.html.erb

I'd like to clean up this view. You can make your kits settings exactly like these files including margin, padding, ect.